src/components/LikeButton/LikeButton.tsx

```diff
@@ -10,42 +10,25 @@ export interface LikeButtonProps extends TPAComponentProps {
   checked: boolean;
   onChange?(event: OnChangeEvent): void;
   disabled: boolean;
+  animation: boolean;
```

Comment:
I think the `LikeButton` should be animated by default.
So there's no need to add a prop to it, since we don't want users to disable the animation.